wisembly/amqp-bundle/1.3/Makefile

```diff
@@ -0,0 +1,16 @@
+CONSOLE=bin/console
+
+ifndef GATE
+gate:
+	$(error GATE not set)
+endif
+
+gate:
+
+consume: gate $(CONSOLE)
+	$(CONSOLE) wisembly:amqp:consume $(GATE) $(OPTIONS)
+
+publish: gate $(CONSOLE)
+	$(CONSOLE) wisembly:amqp:publish $(GATE) "$(MESSAGE)"
```

Comment:
It's used as in `make consume GATE=my-gate` or `make publish GATE=my-gate MESSAGE=""`, but I also thought it would be overkill (the `GATE` env var is required though).

This is the only way I could find to use "arguments" in the Makefile, but then you are probably right by saying that this would be overkill, and better to directly use the command instead... but then, should in the Makefile only be commands without required arguments ?